docker/Dockerfile

```diff
@@ -50,5 +50,8 @@ RUN find /workspace/TensorRT-Model-Optimizer -name "requirements.txt" | while re
     pip install -r "$req_file"; \
     done
 
+# Precompile quantization extensions since this may take several minutes on every docker image restart so it's best to do it once ahead of time
+RUN python -c "import modelopt.torch.quantization.extensions as ext; print(ext.cuda_ext); print(ext.cuda_ext_fp8)"
```

Comment:
The right way of doing this would be 

```
RUN if [ -z "$TORCH_CUDA_ARCH_LIST" ]; then \
    export TORCH_CUDA_ARCH_LIST="5.2 6.0 6.1 7.0 7.2 7.5 8.0 8.6 8.7 9.0+PTX"; \
    fi
RUN python -c "import modelopt.torch.quantization.extensions as ext; print(ext.cuda_ext);" \
```